resolwe_bio/descriptors/general_clinical.yml

```diff
@@ -0,0 +1,1135 @@
+- slug: general-clinical
+  name: General clinical annotation
+  description: General clinical metadata schema.
+  version: 1.0.0
+  schema:
+    - name: general
+      label: General
+      group:
+        - name: species
+          label: Species
+          type: basic:string
+          default: Homo sapiens
+          required: false
+        - name: annotator
+          label: Annotator
+          type: basic:string
+          required: false
+    - name: subject_information
+      label: Subject information
+      group:
+        - name: sample
+          label: Sample
+          type: basic:string
+          required: false
+        - name: subject_id
+          label: Subject ID
+          type: basic:string
+          required: false
+        - name: batch
```

Comment:
We often use non-integer batches, e.g. 1a, 1b, etc. Consider changing the type to `basic:string`.